created test rmds to aid YJ rule development

JacksonJ-KC opened the pull request.

yunjoonjung-PNNL commented:
Thank you! I'll make use of these TCDs and if I have any questions, I'll get back to this PR to ask questions.

After using these TCDs, I'll approve this PR.

yunjoonjung-PNNL commented on rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_22.json:
```diff
@@ -0,0 +1,230 @@
+                                                        }
+                                                    ],
+                                                    "air_energy_recovery": {
+                                                        "energy_recovery_operation": "WHEN_MINIMUM_OUTSIDE_AIR"
```
Could you add ```id``` (something like ```"id": "Air Energy Recovery 1",```)? The ```id``` key is a required key and without it the TCD is not valid against the schema.

yunjoonjung-PNNL commented on rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_22.json:
```diff
@@ -0,0 +1,230 @@
+                                                            "id": "Return Fan 1"
+                                                        }
+                                                    ],
+                                                    "air_energy_recovery": {
```
Could you also add id (something like "id": "Air Energy Recovery 1",) here?

yunjoonjung-PNNL commented on rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_15.json:
```diff
@@ -0,0 +1,338 @@
+                                                ],
+                                                "design_thermostat_heating_setpoint": 65,
+                                                "spaces": [
+                                                    {
```
Could you add ```id``` here?

yunjoonjung-PNNL commented:
I tried to test out Rule 19-15 and found a few things in the TCD.

1. ```id``` is missing under the ```spaces``` keys.
2.  ```id``` is missing under the ```fan_system``` keys.
3.  ```id``` is missing under the ```supply_fans``` keys.

You can also validate the TCD (instance) against the schema using the ```run_json_schema_validation.py``` developed by Juan. Please let me know if you have any questions!

JacksonJ-KC commented:
> I tried to test out Rule 19-15 and found a few things in the TCD.
> 
> 1. `id` is missing under the `spaces` keys.
> 2. `id` is missing under the `fan_system` keys.
> 3. `id` is missing under the `supply_fans` keys.
> 
> You can also validate the TCD (instance) against the schema using the `run_json_schema_validation.py` developed by Juan. Please let me know if you have any questions!

Thanks for the reminder! It had been a couple of months since I last worked on TCDs. I will validate moving forward.
Added ids to 'spaces'. Removed references to 'fan_system' and 'supply_fans' and replaced with terminals[0].fan parameters, as this is testing terminal systems.

yunjoonjung-PNNL commented on rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_15.json:
```diff
@@ -0,0 +1,332 @@
+            "schema_version": "0.0.23"
+        },
+        "rmr_transformations": {
+            "baseline": {
```
Could we have the ```proposed```?

JacksonJ-KC replied:
Added

yunjoonjung-PNNL commented on rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_15.json:
```diff
@@ -0,0 +1,550 @@
+                                        ],
+                                        "heating_ventilating_air_conditioning_systems": [
+                                            {
+                                                "id": "System 10",
```
Could you double-check this system? When I used the ```get_baseline_system_types``` function, it says "Error: HVAC system 10 does not match any baseline system types.".

JacksonJ-KC replied:
should be resolved now, let me know if not

yunjoonjung-PNNL commented on rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_15.json:
```diff
@@ -0,0 +1,550 @@
+                                                    "fan_control": "CONSTANT",
+                                                    "supply_fans": [
+                                                        {
+                                                            "id": "Supply Fan 1"
```
Could you add the ```is_airflow_automated``` key? This is required in the ```are_all_hvac_sys_fan_objs_autosized```.

JacksonJ-KC replied:
added
